<launch>
  <node name="prm_halton_planner" pkg="master_project" type="prm_halton_planner.py" output="screen" ns="prm_halton" > 
    <param name="max_nodes" value="1000"/>
    <param name="robot_radius" value="0.5"/>
    <param name="max_connection_distance" value="2.0"/> 
  </node>
</launch>